### ISSUE TITLE
Preview is not shown in default project for Python Django stack

### ISSUE THREAD

**Issue by Katka92** [eclipse-che/che#15053]
### Describe the bug
When the default project in `Python Djnago` stack is built, the web page is opened but the page is not here.

### Che version

 - [x] latest - 7.3.1
 - [ ] nightly
 - [ ] other: please specify

### Steps to reproduce

1. Create `Python Django` stack with default project
2. Run the application (run commands: install dependencies, migrate, run server)
3. The page is opened but not loaded correctly

### Expected behavior
The page should be loaded correctly.

### Runtime
   - [ ] kubernetes (include output of `kubectl version`) 
   - [ ] Openshift (include output of `oc version`)
   - [x] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Screenshots
![Screenshot from 2019-10-31 16-26-25](https://user-images.githubusercontent.com/29278800/67961392-efcc0880-fbfb-11e9-9df1-3a589d433f14.png)

### Installation method
   - [x] chectl 
`server:start --platform minikube -i eclipse/che-server:7.3.1 -t ${WORKSPACE}/deploy --k8spodreadytimeout=180000 --listr-renderer=verbose` where the ${WORKSPACE} is a path to Che with 7.3.1 tag checkouted.
   - [ ] che-operator 
   - [ ] minishift-addon
   - [ ] I don't know

### Environment
   - [ ] my computer 
       - [ ] Windows
       - [x] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify) 
   - [ ] other: please specify

**Comment by RomanNikitenko** [comment]
![preview_bug](https://user-images.githubusercontent.com/5676062/67969047-1cd6e600-fc11-11e9-9285-5559dd1a5649.png)

**Comment by tolusha** [comment]
It is the case when web page doesn't work in preview and has to be opened in a separate tab.

**Comment by schwingkopf** [comment]
I get the exact same error, in preview as well as separate tab in firefox.

**Comment by ScrewTSW** [comment]
@tolusha Yes. It works outside of the preview, but because the site returns `404` HTML code, the Theia Preview just shows the usual 404 screen, even though the site also returns web page content.

I can see the proper web site in firefox and chrome, but Theia just has the blank 404 screen

**Comment by ScrewTSW** [comment]
I think there is also a `same-origin` policy set in the example which prevents the page to be loaded inside an iframe

**Comment by ibuziuk** [comment]
Closing based on the area label since https://github.com/eclipse-che/che-devfile-registry was archived in favour of https://registry.devfile.io/
